.travis.yml

```diff
@@ -65,7 +65,7 @@ matrix:
       env:
         - DESC="Releasenotes generation"
         - CMD="./.ci/travis/travis.sh releasenotes-gen"
-        - SKIP_CI="false"
+        - SKIP_CI_FOR_FILES="false"
```

Comment:
Please update name in all other CI configs. To keep name consistent.
https://github.com/checkstyle/checkstyle/search?utf8=%E2%9C%93&q=SKIP_CI&type=